dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorCode.java

```diff
@@ -132,6 +132,12 @@
     // TODO See TODO on error usage
     E1047( "Event: `{0}`, date belongs to an expired period. It is not possible to create such event." ),
 
+    E4000( "Relationship: `{0}` cannot link itself" ),
```

Comment:
nitpick: I think "cannot link to itself" or "cannot have a link to itself" is more correct

Reply:
Fixed